staff/Melinx/js/challenges/vendor/jasmine/3.1.0/spec/PlayerSpec.js

```diff
@@ -0,0 +1,58 @@
+describe("Player", function() {
```

Comment:
spec/ folder should not be pushed. they are files just explaining an example in jasmine, but not part of jasmine itself, the library. please, delete spec/ folder, and push again!